File: Dtos/MonitoringDto.cs
using MtpApp.Models;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Web;

namespace MtpApp.Dtos
{
    public class MonitoringDto
    {
        public int Id { get; set; }
        [Required]
        [StringLength(50)]
        public string MonitoringTime { get; set; }
        [Required]
        [StringLength(255)]
        public string Enroll { get; set; }
        [StringLength(255)]
        public string Type { get; set; }
        [Required]
        public int ClientId { get; set; }
        public Client Client { get; set; }
        [Required]
        public DateTime MonitoringDate { get; set; }
        [Required]
        public DateTime NextMonitoringDate { get; set; }
        [Required]
        [StringLength(255)]
        public string Monitoringtype { get; set; }
        public int? PlacementId { get; set; }
    }
}

File: spring-backend/src/main/java/com/mtp/api/dto/MonitoringDto.java
package com.mtp.api.dto;

import lombok.Data;
import java.time.LocalDateTime;

@Data
public class MonitoringDto {
    private Integer id;
    private String monitoringTime;
    private String enroll;
    private String type;
    private Integer clientId;
    private String clientName;
    private LocalDateTime monitoringDate;
    private LocalDateTime nextMonitoringDate;
    private String monitoringtype;
}
